GrandCentralBoard/Widgets/Slack/SlackMessagesWidget.swift

```diff
@@ -11,11 +11,7 @@ import GCBCore
 
 final class SlackMessagesWidget: WidgetControlling {
 
-    private let widgetView: UILabel = {
-        let label = UILabel()
-        label.textColor = UIColor.whiteColor()
-        return label
-    }()
+    private let widgetView = MessageBubbleView()
```

Comment:
Yes, the widget shows only one last message with '@here' or '@channel'. It is showing only the bubble now, in the next PR I will add an avatar of the person who wrote the message.